.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,93 @@
+name: "🐛 Bug report"
+description: Report a bug found while using the Berty apps or packages.
+labels: ["bug", "Ready for Review"]
+assignees:
+- peletron
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: dropdown
+    id: product
+    attributes:
+      label: Berty product
+      description: What Berty product are you seeing the problem on?
+      options:
+        - Mobile app
+        - Desktop app
+        - CLI tools (berty mini, daemon, etc.)
+        - Code package
+    validations:
+      required: true
+  - type: input
+    id: product-version
+    attributes:
+      label:  Berty product version
+      description: What is the version of this Berty product?
+      placeholder: e.g. 2.396.1
```

Comment:
Now the tip is missing as mentioned in my previous comment. https://github.com/berty/berty/pull/4191#discussion_r904027170